src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManager.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.command.gui;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.observer.Publisher;
+
+public interface CommandManager {
+
+	void clearCurrentCommand();
+	void runCommand();
+	String getCurrentCommandString();
+	void setCurrentCommand(DriverCommand commandList);
+	Publisher getChangePublisher();
+	void setDriverManager(DriverManager driverManager);
```

Comment:
DriverManager dependency is still here. Is this method so important to add it to the interface?